### DISCUSSION TITLE
New condition for entry fields: only show entries owned/editable by the user

### DISCUSSION THREAD

**Discussion by darwinsoftware** [craftcms/cms#11995] [Ideas] [answered]
We have a Craft website for a healthcare clinic and we are introducing CMS user accounts so that the practitioners in the clinic can modify their profiles (channel) on the site, as well as publish and link their own articles (channel) to their profile. 

We've introduced a permissions group for these CMS users, giving them the ability to edit only their own profiles and articles. On their profile entry, there is an entry field that allows them to select which articles they feature on their profile. 

The problem is that all articles are shown on this linked entries field, not just the articles owned/editable by the user. We don't want them to see other practitioners' articles and certainly don't want them to be able to feature these articles on their own profiles!

To me, this indicates that it would be useful to have a checkbox when creating an entry field in the CMS, such that only entries owned and/or editable by the user are displayed and available for selection.

**Comment by brandonkelly** [comment, marked as answer]
Added “Editable” condition rules for asset and entry conditions, for Craft 4.3 🎉 (#12048)

**Reply by brandonkelly** [reply]
Craft 4.3 was released earlier this week, with this change!

**Reply by darwinsoftware** [reply]
@brandonkelly could we get a "Viewable" condition as well, for when a user only has permission to view an entry but not save/edit it?

**Reply by brandonkelly** [reply]
@darwinsoftware Can you create a new discussion for that?

**Reply by darwinsoftware** [reply]
Roger that; https://github.com/craftcms/cms/discussions/12240
